automobile/highway-overtake.md

```diff
@@ -0,0 +1,11 @@
+## Highway overtake
+
+%figure "The Highway overtake world"
+
+![highway_overtake.png](images/highway_overtake.png)
+
+%end
+
+This world represents a simple straight highway with 4 lanes (including one emergency lane).
+A dense traffic is added to the highway thanks to the SUMO interface.
+The car will try to overtake the other vehicles using distance sensors, and a supervisor is used in order to display the radar targets seen by the car's front radar.
```

Comment:
The car &rarr; A car
I would remove the future form ("will try" &rarr; "tries").